src/plugins/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@ set(plugin_sources gridDynPlugins.cpp)
 
 set(plugin_headers gridDynPluginApi.h gridDynPlugins.h)
 
-add_library(pluginLibrary ${plugin_sources} ${plugin_headers})
+# TODO static?
+add_library(pluginLibrary STATIC ${plugin_sources} ${plugin_headers})
```

Comment:
Should this be static? Nearly everything else is

Reply:
The plug in library is kind of an old experiment,  this should probably be static,   I was trying to make a plug in capabable library for GriDyn so Models could be loaded on dynamically. as needed.  I realized it wasn't going to work the way I wanted it to so I stopped working with it.  I think the C shared library needs to be a little more developed before attempting again since I think the models in a plugin would have to be built using a C based interface and I think we need to understand how to make those models properly before going in this direction again.  So the option to enable this should probably be hidden for the time being since it won't do anything as is.